gm4_better_fire/data/gm4_better_fire/functions/on_fire.mcfunction

```diff
@@ -1,5 +1,8 @@
+# @s, flame arrow or creeper on fire
+# runs from main
+
 #fire arrows place fire
-execute at @s[type=arrow] run setblock ~ ~ ~ fire keep
+execute as @s[type=arrow,tag=!gm4_cb_arrow,nbt={inGround:1b}] if data entity @s Fire at @s run function gm4_better_fire:arrow
```

Comment:
I can't find any reference to this `gm4_cb_arrow` tag.

Reply:
#615 adds a common tag to all arrow-based projectiles used in Crossbow Cartridges.  Unknown if torches shot using Crossbow Cartridges should activate Better Fire.

Reply:
Instead of checking for the crossbow cartridges tag, better fire should add a general ignore tag which CC should set.